openfasoc/MLoptimization/eval.py

```diff
@@ -0,0 +1,118 @@
+#training import
+import numpy as np
+import gym
+import ray
+import ray.tune as tune
+from ray.rllib.algorithms.ppo import PPO
+from run_training import Envir
+from ../generators/gdsfactory-gen/sky130_nist_tapeout import single_build_and_simulation
```

Comment:
Directory names have changed.
At this point the ML module is a one-off business, and the API is not well established (compared to GLayout), in the long run this kind of project-specific hack should be avoided.